docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
"Debian Sid (May 6th)" is that the best you can do? Seems like a pretty fuzzy/unhelpful reference point for "the version that these docs last worked on."